src/Control/SimpleSearchField.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+namespace ipl\Web\Control;
+
+use ipl\Html\Attributes;
+use ipl\Html\Form;
+use ipl\Html\FormElement\InputElement;
+use ipl\Html\FormElement\SubmitElement;
+use ipl\Html\HtmlElement;
+use ipl\Web\Url;
+
+/**
+ * Form for simple value based search
+ */
+abstract class SimpleSearchField extends Form
+{
+    protected $defaultAttributes = [
+        'class'                 => 'icinga-form icinga-controls search-field',
```

Comment:
Remove `icinga-form` and `icinga-controls`. These classes are from Icinga Web 2.